src/group_theory/subgroup.lean

```diff
@@ -723,6 +737,21 @@ lemma map_supr {ι : Sort*} (f : G →* N) (s : ι → subgroup G) :
 lemma comap_inf (H K : subgroup N) (f : G →* N) : (H ⊓ K).comap f = H.comap f ⊓ K.comap f :=
 (gc_map_comap f).u_inf
 
+@[to_additive] lemma comap_sup
+  (H K : subgroup N) (f : G →* N) : comap f H ⊔ comap f K ≤ comap f (H ⊔ K) :=
+sup_le (comap_mono le_sup_left) (comap_mono le_sup_right)
+
+@[to_additive] lemma map_inf (H K : subgroup G) (f : G →* N) :
+  map f (H ⊓ K) ≤ map f H ⊓ map f K :=
+le_inf (map_mono inf_le_left) (map_mono inf_le_right)
+
+@[to_additive] lemma map_inf_eq (H K : subgroup G) (f : G →* N) (hf : function.injective f) :
+  map f (H ⊓ K) = map f H ⊓ map f K :=
+begin
+  rw ← set_like.coe_set_eq,
+  simp [set.image_inter hf],
+end
+
```

Comment:
You've put these lemmas in a place that separates `comap_inf` from `comap_infi` - can you move those two lemmas to be adjacent again?